ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml

```diff
@@ -5283,13 +5283,6 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-4.13-upgrade-from-stable-4.13-aws-ipi-ovn-hypershift-control-plane-p1-f4
-  reporter_config:
```

Comment:
I think those removal are unexpected. Would you mind to double confirm ?

Reply:
`make jobs ` need more options? these lines were removed after `make jobs `. @liangxia